index.html

```diff
@@ -0,0 +1,63 @@
+            <a id="lets_connect" href="https://www.linkedin.com/in/dara-adesoji-485a1699/">LET’S CONNECT</a>
+            <div class="social_icons">
+                <a href="https://www.twitter.com/dara_soji">
+                    <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABgAAAAYCAYAAADgdz34AAAAAXNSR0IArs4c6QAAAUdJREFUSEvdleExBEEQhb+LgAzIABEgAkTgRIAIEAEhyAARkAGXAREgAuq7mrmanZuZ29uq/XNTdX92ut/r97qnb8LIZzIyPptHsA38rLDNmEvgFNgHPoFH4C7k+f05YuQWGWzAR4PkDTgs3Jsj4TXwUCLw8j0oOAMEyk+MqfFLMgsE8yJzBX9JplUoO7VMdU8Ndb/BgUVxOYHenWQAfvOnfQfAfYPgJRAsQlICm7cbwHYGvg8V36a5OcH3QOCY1mlwqQfaMLR68bSwM4F5D5wSG7Q1QMlXsLiTWloVTsp53qwehEv2lCzy2xHw2gMwDbF61S9tgdqycxJu1iA5rjzM5jZ1bH1s2tU6F2EXFWNqCrQpLrQauK/2qgWe90BQK97rYY0vVnDHunlKCqah0TYtkrnAbGC6NlZhz+837x+tl+x1gka36B/CnzgZFPaBJAAAAABJRU5ErkJggg==" alt="twiter logo"/>
+                </a>
+                <a href="https://www.linkedin.com/in/dara-adesoji-485a1699/">
+                    <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABgAAAAYCAYAAADgdz34AAAAAXNSR0IArs4c6QAAAP5JREFUSEvtlc8NAUEUh7+tgBJw4IgK0IESnCUOEgVQgEQkzkoQHdABNwcHOqAB5MnbZHYysXbGxmXnNPt23u97/2Y3IucV5ayPC1ADHsDlF3AbsARGKjwHJqEQE9AATpZgBbiGQNIA1dBS2SVaAUONeAGMQ6IXX1eT69rkc6i4C9AFnip8Bw66bwEly14GmoD4yLmjq5x2BrG4aO3VWfY7oKMAsU+BDSAQc4l9Zhp8AHI/RNgWj3V7GtD72QeQ1pot0I8P+QJMkQGwNqiSoYx3UAZtYwBE6GYMQaIyvhnYfuYQFABn74oSJT4Vrgv6dYnSbmnm93/56WeO8pPDC6yAShkkBxiqAAAAAElFTkSuQmCC" alt="linkedin logo"/>
+                </a>
+                <a href="https://github.com/dara-soji">
+                    <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABgAAAAYCAYAAADgdz34AAAAAXNSR0IArs4c6QAAAZdJREFUSEu1lYExBVEMRe+vABWgAlSAClABKkAFqAAdUMGnAlSADuiACpgzk+zkZ7L735rZzOzs7puXe5P7kryZJrbZxPhqITiUdCBp2x5ierfnRdLDUJBDBADfSNpYkuWnpAtJj9W+PoIrSZcj5bs1ogW3iuA/4A56J+k8MmSCPUnPtuFJEmQnks5s7cPeW/ZG/3vbc2xrR1GuTICe67bx2gj4XZX0nSSLazFrMDZ9byTgUOcBhOiIvsWyrKeW2UKZkqqnCei+JMqwxSjhtyq4mAFgu0FrnMZY9KdPdnCOBGi8YoivkjjwMRYJOuxI8BvQJiGIEZDN2pjwrcpcgS+fADGDfMi0P93ZYjQXY8Wtq8C+Mv2x86D86M7cAw5EL1B5OZCyTHGiSUiTEiUqL1u6mp5wIoDJmKrjO1onT64i/n1UAESZOUjsagfLkvr6Qv9Uw871LKdjCLUaikuHnfs7CQ1DNj7UohSZoMpy8EZDLmRg+FXOToDmnE85VlquTIjIgmyiMUq47cqbzDe2ELT0Qe+eyQn+AIklVhnz1DvpAAAAAElFTkSuQmCC" alt="github logo"/>
+                </a>
+                <a href="https://www.twitter.com/dara_soji">
+                    <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABgAAAAYCAYAAADgdz34AAAAAXNSR0IArs4c6QAAAQhJREFUSEvtldERwUAURU8qoAN0oAM6QAdUgApQASqgBDqgAyUogQqYa3bN2tlkk4wYH7k/ycjuPfvue4mEipVU7E8NiCbsRjQGdsANWALb6O7PBdq/Nj/Ngb3uXcAVaDl7DsDEALNYTWMsgJW8Oj7gEXDRwhFwSSF0TdW6+nod3q0gBLCbZoHIbCSqIKRCABnYyHSvXg0jPSoMkJ8ik9o5BqAUIIfve0kUcAT6QCPiegdOwMBbFwWsgI3JvpcCOZteaAgWZQB64SRdfQMdIOt5rgqsgSCKS5MkaYIUi1XoAIUBMrMzr8+Jq68B0vr9HwBNzNQ5otvE2Pz7FehLrMmq/9Fiyf0goide8TgZZ2khYwAAAABJRU5ErkJggg==" alt="gmail logo"/>
+                </a>
+            </div>
```

Comment:
- [ ] 🚧 You did a good job using links for your social media! 💯 However, the markup for this component doesn't provide a lot of information to the browser or assistive technologies as the `<div>` tag is just a generic element. Since this represents a "List" of social media icons, you could use the `<ul>` tag which describes an unordered list, and you can wrap each icon inside a `<li>` tag which is for list items. 🤓 🔧